testbed/tests/app/test_desktop.py

```diff
@@ -168,159 +170,183 @@ async def test_menu_minimize(app, app_probe):
     assert window1_probe.is_minimized
 
 
-async def test_full_screen(app, app_probe):
-    """Window can be made full screen"""
-    window1 = toga.Window("Test Window 1", position=(150, 150), size=(200, 200))
-    window2 = toga.Window("Test Window 2", position=(400, 150), size=(200, 200))
-
-    window1_widget = toga.Box(style=Pack(flex=1))
-    window2_widget = toga.Box(style=Pack(flex=1))
-    window1_widget_probe = get_probe(window1_widget)
-    window2_widget_probe = get_probe(window2_widget)
-
-    window1.content = toga.Box(
-        children=[window1_widget], style=Pack(background_color=REBECCAPURPLE)
+async def test_presentation_mode(app, app_probe, main_window, main_window_probe):
+    """The app can enter presentation mode."""
+    window_information_list = list()
+    windows_list = list()
+    for i in range(len(app.screens)):
+        window = toga.Window(title=f"Test Window {i}", size=(200, 200))
+        r = random.randint(0, 255)
+        g = random.randint(0, 255)
+        b = random.randint(0, 255)
```

Comment:
Random behavior isn't desirable, as it becomes difficult to work out what changes are intentional. Cycling between four colors (CORNFLOWERBLUE, FIREBRICK, REBECCAPURPLE, GOLDENROD would be the obvious set) will cover people with up to 4 monitors... which I warrant will be almost everyone.

Reply:
I have changed it.